docs/source/release/v6.2.0/diffraction.rst

```diff
@@ -5,110 +5,117 @@ Diffraction Changes
 
 Single Crystal Diffraction
 --------------------------
 New features
 ############
-- New algorithm :ref:`HB3AIntegrateDetectorPeaks <algm-HB3AIntegrateDetectorPeaks>` for integrating four-circle data from HB3A in detector space.
 - New algorithm :ref:`ApplyInstrumentToPeaks <algm-ApplyInstrumentToPeaks>` to update the instrument of peaks within a PeaksWorkspace.
-- New plotting script that provides diagnostic plots of SCDCalibratePanels output.
-- New plotting script that provides diagnositc plots of SCDCalibratePanels2 on a per panel/bank basis.
-- Exposed :meth:`mantid.api.IPeak.getCol` and :meth:`mantid.api.IPeak.getRow` to python
-- Added two integration methods to :ref:`HB3AIntegrateDetectorPeaks <algm-HB3AIntegrateDetectorPeaks>` for simple cuboid integration with and without fitted background.
 - New algorithm :ref:`ConvertPeaksWorkspace <algm-ConvertPeaksWorkspace>` for quick conversion between PeaksWorkspace and LeanElasticPeaksWorkspace.
-- New definition file for D19 ILL instrument added.
 - New algorithm :ref:`FindGlobalBMatrix <algm-FindGlobalBMatrix>` that refines common lattice parameters across peak workspaces from multiple runs with a different U matrix (which encodes the orientation) per run.
+- New algorithm :ref:`HB3AIntegrateDetectorPeaks <algm-HB3AIntegrateDetectorPeaks>` for integrating four-circle data from HB3A in detector space.
+- New plotting script that provides diagnostic plots of :ref:`SCDCalibratePanels <algm-SCDCalibratePanels-v2>` on a per panel/bank basis.
+- Exposed :meth:`mantid.api.IPeak.getCol` and :meth:`mantid.api.IPeak.getRow` to python.
+- Added two integration methods to :ref:`HB3AIntegrateDetectorPeaks <algm-HB3AIntegrateDetectorPeaks>` for simple cuboid integration with and without fitted background.
+- New definition file for D19 ILL instrument added.
 
 Improvements
 ############
-- Find detector in peaks will check which det is closer when dealing with peak-in-gap situation for tube-type detectors.
-- Existing :ref:`SCDCalibratePanels <algm-SCDCalibratePanels-v2>` now provides better calibration of panel orientation for flat panel detectors.
-- Existing :ref:`DGSPlanner <dgsplanner-ref>` expanded to support WAND²
-- Existing :ref:`MaskPeaksWorkspace <algm-MaskPeaksWorkspace-v1>` now also supports tube-type detectors used at the CORELLI instrument.
-- Existing :ref:`SCDCalibratePanels <algm-SCDCalibratePanels-v2>` now retains the value of small optimization results instead of zeroing them.
-- Existing :ref:`IntegrateEllipsoids <algm-IntegrateEllipsoids-v1>` now can use a different integrator for satellite peaks.
+
+- Existing :ref:`DGSPlanner <dgsplanner-ref>` expanded to support WAND².
+- Existing algorithm :ref:`IntegrateEllipsoids <algm-IntegrateEllipsoids-v1>` now can use a different integrator for satellite peaks.
```

Comment:
Could you put the two notes relating to IntegrateEllipsoids adjacent to each other?